pyproject.toml

```
@@ -26,7 +26,7 @@ requires = [
     "cachetools==5.2.0",
     "importlib-metadata==4.12.0",
     "evaluate",
-    "datasets==2.6.1",
+    "datasets==2.6.2",
```

i would suggest being less strict with this pin

```suggestion
    "datasets>=2.6",
```